src/main/java/uk/nhs/hee/tis/trainee/forms/service/LtftService.java

```diff
@@ -256,26 +256,71 @@ public Optional<Boolean> deleteLtftForm(UUID formId) {
   }
 
   /**
-   * Approve the LTFT form with the given id.
+   * Submit the LTFT form with the given id.
    *
-   * @param formId The id of the LTFT form to approve.
-   * @param detail The status detail for the approval.
-   * @return The DTO of the approved form, or empty if form not found or could not be approved.
+   * @param formId The id of the LTFT form to submit.
+   * @param detail The status detail for the submission.
+   * @return The DTO of the submitted form, or empty if form not found or could not be submitted.
    */
   public Optional<LtftFormDto> submitLtftForm(UUID formId,
       LtftFormDto.StatusDto.LftfStatusInfoDetailDto detail) {
+    return changeLtftFormState(formId, detail, LifecycleState.SUBMITTED);
+  }
+
+  /**
+   * Unsubmit the LTFT form with the given id.
+   *
+   * @param formId The id of the LTFT form to unsubmit.
+   * @param detail The status detail for the unsubmission.
+   * @return The DTO of the unsubmitted form, or empty if form not found or could not be
+   *         unsubmitted.
+   */
+  public Optional<LtftFormDto> unsubmitLtftForm(UUID formId,
+      LtftFormDto.StatusDto.LftfStatusInfoDetailDto detail) {
+    return changeLtftFormState(formId, detail, LifecycleState.UNSUBMITTED);
+  }
+
+  /**
+   * Withdraw the LTFT form with the given id.
+   *
+   * @param formId The id of the LTFT form to withdraw.
+   * @param detail The status detail for the withdrawal.
+   * @return The DTO of the withdrawn form, or empty if form not found or could not be withdrawn.
+   */
+  public Optional<LtftFormDto> withdrawLtftForm(UUID formId,
+      LtftFormDto.StatusDto.LftfStatusInfoDetailDto detail) {
+    return changeLtftFormState(formId, detail, LifecycleState.WITHDRAWN);
+  }
+
+
+  /**
+   * Change the state of the LTFT form with the given id.
+   *
+   * @param formId      The id of the LTFT form to change.
+   * @param detail      The status detail for the change.
+   * @param targetState The state to change to.
+   * @return The DTO of the form after the state change, or empty if form not found or could not be
+   *         changed to the target state.
+   */
+  protected Optional<LtftFormDto> changeLtftFormState(UUID formId,
```

Comment:
I thought I could sneak my version of this in before you, guess I have some more refactoring to do 😄

Reply:
Accommodating the admin endpoints is what my changes were about, I'll make it work after this is merged